pkg/process/procutil/data_scrubber_windows.go

```diff
@@ -0,0 +1,85 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build windows
+
+package procutil
+
+import (
+	"strings"
+)
+
+// Extensions in which windows defaults the exec commands. Additional extensions can be included. 
+var (
+	winDotExec= []string{
+		".com",".exe",".bat",".cmd",".vbs", ".vbe",".js",".jse",".wsf",".wsh",".psc1", ".ps1"} 
+)
```

Comment:
[nit/minor] you only have one variable here. maybe just go without the parens like  `var winDotExec =  ...`